extensions/wikia/ContentReview/ContentReview.hooks.php

```diff
@@ -123,9 +123,9 @@ public function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
 	 * @throws \MWException
 	 */
 	public function onSkinAfterBottomScripts( $skin, &$bottomScripts ) {
-		global $wgUseSiteJs;
+		global $wgOut, $wgUseSiteJs;
 
-		if ( !empty( $wgUseSiteJs ) ) {
+		if ( !empty( $wgUseSiteJs ) && $wgOut->isUserJsAllowed() ) {
```

Comment:
You can avoid the global since `$skin` is a `ContextSource`, so this could be `$skin->getOutput()->isUserJsAllowed()`.